lib/spack/spack/build_systems/cbi.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from typing import Tuple
+
+import spack.builder
+import spack.directives
+import spack.package_base
+
+from ._checks import BaseBuilder, apply_macos_rpath_fixups
+
+
+class CBIPackage(spack.package_base.PackageBase):
+    """General purpose class with three phases (``configure``, ``build`` and
+    ``install``) that need to be coded by packagers.
+    """
+
+    #: This attribute is used in UI queries that require to know which
+    #: build-system class we are using
+    build_system_class = "CBIPackage"
+
+    spack.directives.buildsystem("cbi")
+
+
+@spack.builder.builder("cbi")
+class CBIBuilder(BaseBuilder):
```

Comment:
What is this?

Reply:
A leftover. Initially I coded this to be a "generic" package with 3 phases (configure, build, install). Since we decided to allow packages to customize phases at least on `GenericBuilder` I removed every reference to this, but forgot to delete this file.